%!TEX root = thesis_cgo.tex
\chapter{Introduction}


\epigraph{Everything existing in the universe is the fruit of chance and necessity.}{Democritus}

Molecular  machines  are  assemblies of proteins and associated molecules that  work together  in a coordinated manner  to solve a biological problem.  These biological problems, such as coordinating chromosome segregation  during cell division, regulation of cell cycle timing, execution of protein  synthesis, etc.  encompass many processes essential to life. Given the necessity for survival in the face of ever changing environments,  evolution has produced a large diversity of molecular mechanisms for solving these problems in a flexible yet robust  manner.  A biological machine that  only functions properly under a narrow range of  conditions is less likely to support the life of a single individual or a population.  Therefore,  at the core of each of those processes are highly complex networks of proteins that  are able to assemble, communicate, coordinate,  self-regulate and self-correct in order to accomplish the necessary task reliably.  For example, the vital process of DNA replication is executed by a large multitude of proteins that  each contribute to the process of copying the genome ~\cite{bell2002dna}. The replication  machinery must first read environmental  cues for initiating  replication  at the correct time. Meanwhile, complex combinatoric  signaling networks ensure that  DNA replication  unfolds in a processive manner,  enzymatic components  perform physical work to unwind DNA strand  for copying, and others  communicate  with the DNA repair machinery  to correct copying errors and avoid harmful mutations. It is clear that  solving the biological problem of DNA replication requires the ability of participating proteins to interact  with many different partners  and mediate  many different processes.


The structure and function of each protein is encoded in its unique sequence, or chain, of amino acids. Physical interactions between amino acids give rise to a specific 3D arrangement of the protein chain, also known as structure. The structure of each protein allows for specific interactions between  proteins  to  assemble molecular machines, recruit  necessary factors and mediate chemical reactions. See \figref{fig:tub4} for a visualization of protein structure. Since the 1950s when the first X-ray crystallography protein structure was solved ~\cite{kendrew1958three}, we have learned a great deal about how 3D architecture and conformational sampling of the chains give rise to protein function. X-ray crystallography accesses atomic-scale conformations of folded protein  domains, allowing us to infer that  coordinated  motions between structural conformations is the main element of control in protein function ~\cite{hegyi1999relationship}. For example, X-ray crystallography experiments have shown that the activity of Calmodulin, an important signalling protein, is modulated by conformational changes of its $\alpha$-helix domains brought about by binding of Ca\textsuperscript{2+} ions~\cite{meador1992target}. These conformational rearrangements initiate a clamping motion of the helical domains which allow Calmodulin to bind to its downstream targets. However, it is important to note that X-ray crystallography only offers static pictures of protein structure, and provides information mostly on the spatial arrangement of relatively large and stable domains. It therefore became a long standing dogma that the stable 3D folds of a protein chain dictate a protein's function, also known as the "one structure - one function" paradigm. 

However, as we saw with DNA replication, a single protein is often required to fulfill many functions, and interact with various different partners. It is therefore unlikely that such large scale and consequently slow structural motions can account for all of the precise and rapid control we observe in biological systems. A static description of proteins is not sufficient to explain the degree of functional flexibility and control that we observe. This leaves us with several questions. If one structure means one function, how can the same protein fulfill multiple functions and engage in many different interactions? How can molecular machines offer such precise control of functionality while counting only on a static architectures? The broad aim of this thesis is therefore to improve our understanding of the physical mechanisms underlying the functional complexity of molecular machines.

\begin{figure}[h!]
\centering
\includegraphics[height=0.4\textheight]{tub4}
\FigureCaption{$\gamma$-Tubulin 3D Structure}{Visual representation of the yeast $\gamma$-Tubulin protein based on the human $\gamma$-Tubulin structure derived from X-ray crystallography~\cite{aldaz2005insights}. The protein is composed of a highly ordered globular domain stabilized by various $\alpha$-helices and $\beta$-sheet domains and measures \SI{2.20}{\nm} in radius of gyration. In red, we see the disordered \gct region which was absent in the crystal structure and was modeled by \texttt{RaptorX} ~\cite{kallberg2012template}.}
\label{fig:tub4}
\end{figure}

\section{Disorder in proteins}

In recent years, it has been recognized that functional plasticity  can be found in regions of proteins that  do not adopt stable architectures, also known as intrinsically disordered regions (IDRs). While IDRs are highly flexible and largely unstructured, functional studies have shown that they are necessary for many cellular processes ~\cite{wright2015intrinsically}.  \footnote{Some works make the distinction between IDP and IDR where an IDR is an intrinsically disordered region and IDP is a fully disordered protein.} Instead of relying on a structure for function, IDR functionality lies the absence of structure. This flexibility offers the protein rapid access to a vast pool of conformations with which to fine-tune and diversify its function.  

The study of IDRs is relatively new to structural biology. This is largely due to the fact that the main tool being used for structural biology in the past decades, X-ray crystallography, fails to detect patterns in unfolded chains, making it difficult to study highly dynamic elements in protein such as IDRs. Unstable protein domains such as IDRs that sample many conformations produce averaged out electron scatter patterns that cannot be interpreted ~\cite{putnam2007x}. Techniques that  do produce information  on dynamics,  such as Nuclear Magnetic Resonance (NMR) only developed for proteins  until 1984 ~\cite{wuthrich2001way}, 25 years after the first structure was solved by X-ray crystallography  in 1958 ~\cite{kendrew1958three}.  Another approach for studying protein dynamics with atomic resolution is through computational simulation.  Physical models of proteins whose motions are computed \silico have been shown to provide important information regarding detailed dynamics of biomolecules ~\cite{karplus2002molecular}. However, until recently, these techniques,such as Molecular Dynamics (MD), were greatly limited by shortcomings in computer power. However, with large advances  in experimental  and computational techniques  in recent years, we have been able to study the dynamic properties  of IDPs in great detail, and have found that  they play key roles in the control of molecular machines.

Over 15,000 proteins in the Protein Data Bank have been predicted to contain long disordered regions ~\cite{romero1998thousands}; it is therefore not surprising that  IDPs have also been implicated  in a multitude of cellular processes and disease states ~\cite{uversky2008intrinsically}.  Interestingly, it has been shown that viral proteins use IDPs in their proteins to hijack cellular proteins and use the flexibility of IDPs to mimic host proteins and recruit host cellular machinery in order to propagate.\cite{davey2011viruses} This would suggest that viral proteins use IDPs to make efficient use of their smaller genomes and obtain a greater range of function from the limited number of proteins in their genomes. It is now clear that IDPs, through their lack of structure, are an important adaptive feature that drive the functional complexity and robustness we observe in molecular machines. \\

\section{Physical mechanisms of IDR function in cellular machines}

In this section we will give a brief account of some of the physical mechanisms of IDR function that have been described in the literature. \\

{\it Phosphorylation}

A key aspect of dynamic control is the ability to modulate function in a precise and reversible manner.  The cell needs to be able to induce and inhibit interactions in a time and space dependent manner. To solve this problem, the cell harnesses the structural malleability of IDRs/IDPs by coupling these elements with post translational modifications, most commonly, phosphorylation. Phosphorylation is the reversible covalent addition of a phosphate  group effected by a protein kinase,  which carries a negative  charge to a tyrosine, serine or threonine  amino acid. The reverse reaction is catalyzed by enzymes called phosphatases which remove the phosphate group. The addition of a phosphate group introduces the potential inter and intra-molecular hydrogen bonding which alters the electrostatic environment of the IDP/IDR ~\cite{van1990effect}. This change can in turn bias the stochastic conformational sampling of the IDP/IDR in a particular direction. Because phosphorylation is reversible, it acts as a means for driving structural switching which can then be used to modulate a large range of interactions ~\cite{kern1999structure, ramelot2001phosphorylation, oxley2008integrin}. Not surprisingly, it has been seen in many studies that IDPs are prime targets for phosphorylation ~\cite{iakoucheva2004importance}.  The use of phosphorylation as an information carrier has been described in various cellular systems. For example, in cell cycle control, there is a strong need for a specific temporal sequence of interactions to be enforced ~\cite{yoon2012cell}. In such a case, the sequential phosphorylation of a single target modifies the affinity for the same target to the next target in the pathway, ensuring that interactions take place in an ordered manner ~\cite{wright2015intrinsically}. Phosphorylation can also be used to enforce thresholds, where in order to avoid the negative impact of accidental interactions, certain interactions will be blocked until an IDP/IDR has achieved a certain number of phosphorylations ~\cite{nash2001multisite}. Having accumulated enough phosphorylations, a structural rearrangement favours the interaction. 

{\it Disorder-Order transitions}

\todo{clarify entropy-enthalpy}

The best explored physical mechanism of IDP function is the fold-on binding paradigm ~\cite{wright2009linking}. In this case, IDPs in the free form are unstructured, and when they encounter their binding target, they undergo a folding transition (disorder to order) to form a stable complex. The lack of structure in the unbound state allows the the IDP/IDR the necessary flexibility to recognize multiple targets, and it allows binding to be inducible instead of constitutive. A well studied example of this kind of mechanism is the binding of the transcriptional activator protein CREB and its co-activator CBP ~\cite{gianni2012folding}. An IDR in CREB known as KID mediates binding to CBP where upon binding, the IDP folds into a pair of helices. However, this binding process is not favoured spontaneously due to a high entropic barrier. \footnote{Protein folding is an ordering of the protein backbone which typically implies a loss of entropy to the system and is therefore energetically disfavoursed, this is the entropic barrier; $\Delta G = \Delta H - T\Delta S$, where $\Delta H$ is the change in enthalpy, or heat energy available to the system, and $\Delta S$ is the entropy, or degree of disorder/conformational freedom available to the system. However, if the folding process releases sufficient heat ($\Delta H$), the loss in entropy is overcome and the folding reaction is energetically favoured; $\Delta G < 0$. The release in heat by the folding of the protein typically increases the entropy of the surrounding water molecules thus preserving the second law of thermodynamics.}However, when the KID is phosphorylated, the phosphoryl group interacts with CBP by forming hydrogen bonds which result in a negative enthalpic change that compensates for the loss of entropy and thus makes the folding reaction favourable. Because of the inducible nature of this interaction, CBP is able to also interact with other co-factors, which has been reported in the literature \cite{radhakrishnan1997solution}. This is an example of how even though the association state of the IDP is ordered, it is  the intrinsic disorder and entropy of the unbound state which is the key for controlling the interaction. \\


\pagebreak

{\it \lq Fuzzy\rq interactions}

\par Disorder to order transitions are not necessary for IDPs to confer functionality. There are a growing number of examples where IDPs/IDRs are involved in functional interactions while remaining in a disordered state \cite{tompa2008fuzzy}. Such interactions have been labeled with the term \lq fuzzy\rq as they maintain a heterogenous conformational ensemble throughout their lifetimes. There exist various physical mechanisms by which fuzziness, or disorder, in binding interactions confers advantages to protein function. For example, binding interactions between an IDP/IDR and a target protein where the IDP/IDR is able to form alternate contacts with its binding target can help reduce the entropic cost of binding, as well as control the accessibility of different sites on the protein for modulating interactions with different targets \cite{graham2001tcf4, fontes2000structural}. IDPs/IDRs can also play a role in interactions without making direct contacts with the binding partner by acting as flexible linkers for folded domains \cite{bhattacharyya2006ste5}, or as \lq antennae\rq for \cite{sigalov2004homooligomerization} recruiting further interactiors and stabilizing the binding of folded domains through long range interactions \cite{zor2002roles, yu1994structural} 

It is becoming increasingly evident that nature has harnessed disorder as an adaptive mechanism for control in protein-function. High flexibility in protein conformational state allows switch-like control over interactions and activity, fine-tuning the kinetics of interactions, precise signal integration, controlled multiple partner binding, etc.  It is due to these biophysical properties that the cell is able to carry out its complex tasks with such robustness and precision. Advances in this field have caused us to reconsider the  \lq one structure, one function\rq paradigm that has prevailed in structural biology for decades. However, this remains a relatively novel area of structural biology, and there still remain many unsolved physical mechanisms in IDPs/IDRs.

\section{IDP function in the mitotic spindle}
 
In this section we will address the role of IDRs in controlling the function of the mitotic spindle. The mitotic spindle is a complex molecular machine composed of microtubules, force generators, chromosomes, and numerous effector molecules which act in a coordinated manner to accomplish the process of chromosome segregation during cell division ~\cite{karsenti2001mitotic}. This process ensures that genetic material is transferred from the mother to the daughter cell in a timely manner, and without errors which would in most cases result in fatality. Because cell division is a fundamental task in every cell's lifetime, mitotic spindles share common design features throughout eukaryotes ~\cite{kubai1976evolution}. The  task  of properly  arranging  and  segregating  chromosomes  is effected ultimately  by hollow \SI{25}{\nm} filaments composed of polymerized tubulin, known as microtubules,  which attach to chromosomes and exert forces to move the  DNA into mother and daughter  cells ~\cite{kline2004mitotic}.  In order to study the underlying mechanisms at play, we work with the mitotic spindle of the budding yeast {\it Saccharomyces cerevisiae} due to its minimal, yet highly conserved construction ~\cite{kubai1976evolution}. 

\subsection{Microtubules}

Microtubules are constructed of alternating pairs, or heterdimers, of the globular proteins $\alpha$ and $\beta$ tubulin. A cylindrical  arrangement  of tubulin  dimers results  in the  formation  of a microtubule, whose rigidity can be adapted in a length dependent manner.   By adding or losing subunits, microtubules can increase and decrease in length, and push or pull directly on targets.  Microtubules also bind proteins that link them to chromosomes, to vesicles, and even to each other when forming microtubule bundles.  ~\cite{dogterom2005force}. Microtubules  have a number  of other  functions,  such  as serving as roadways  along which transport molecules can carry cargo, and form an adaptable “cytoskeleton” that contributes to cell shape and movement. The spontaneous assembly of tubulin dimers in solution into a microtubule is heavily disfavoured. However, when free floating tubulins encounter pre-formed “nucleus” or microtubule seed, the growth of a microtubule is greatly facilitated  ~\cite{joshi1992gamma}. In cells, this template is known as the $\gamma$-Tubulin Ring Complex ($\gamma$-TuRC) which is a ring-like assembly of $\gamma$-Tubulin molecules held together by various other proteins known as $\gamma$-tubulin ring proteins (GRIPs) ~\cite{kollman2011microtubule}. $\gamma$ tubulin shares a similar structure to $\alpha$ and $\beta$ tubulin and have been shown to act as nucleation templates for microtubules when assembled in a ring complex ~\cite{kollman2011microtubule}. 

\subsection{$\gamma$-Tubulin \& the $\gamma$-CT}

Microtubule nucleation was thought to be the sole function of $\gamma$ tubulin for many years. However, recent evidence from budding yeast suggests that $\gamma$-tubulin might have additional functions in controlling microtubule properties ~\cite{vogel2000carboxy,vogel2001phosphorylation, cuschieri2006gamma, nazarova2013cdk1}. $\gamma$-tubulin bound to the spindle poles is phosphorylated in vivo at  8 sites ~\cite{vogel2001phosphorylation,keck2011cell}.
Several functional studies following up on the finding that $\gamma$-tubulin is regulated show that mutations altering phosphorylation sites, all of which lie in IDRs, have consequences on the organization and stability of microtubules but no effect on their nucleation. This opens a relatively unexplored field of functional coupling whereby the $\gamma$-TuRC acts not only as a microtubule nucleator, but also as a signal integration hub for regulating downstream events in microtubule organization.

One phosphorylation site in \tub that has a important role in spindle function is the highly conserved  tyrosine  (Y)  445  which lies in the  disordered  carboxyl  terminal  tail  of \tub  (\gct). The \gct{} is defined as the final 35 residues in the C-terminal portion of $\gamma$-tubulin, which lies outside of the folded globular domain. The  \gct is essential for survival in budding yeast  ~\cite{vogel2000carboxy}. Substitution of an aspartic or glutamic acid (D/E) residue in the place of Y445 results  in slow growing cells with unstable  and misaligned mitotic  spindles ~\cite{vogel2001phosphorylation}. The Y445D/E substitutions are used as a means for constiuatively mimicking the electrostatic  environment  of a phosphate  group  by introducing  a negative  charge. Defects in spindle function observed in these mutants suggest that phosphorylation is limited to a specific stage of the cell cycle and/or subset of molecules. However, very little  is known about  the interactions and physical mechanisms that  phosphorylation  of  Y445 may control.   The  coupling of post-translational modifications  (PTMs) to the C-terminal  tails of tubulins  is well described in the $\gamma$-tubulin orthologues  $\alpha$ and $\beta$ tubulin.  Specific combinations  of PTMs  on the $\alpha$ and $\beta$ tubulin  tails act as a  \lq tubulin code\rq for selectively recruiting  motor  proteins  and  microtubule  associated  proteins to the microtubule  lattice.  A similar code has not yet been described for $\gamma$-tubulin despite evidence that  is is regulated \vivo.  While the functional importance  of phosphorylation and IDRs in $\gamma$-tubulin is becoming increasingly clear, the physical mechanisms  by which local modifications  at  IDRs can have global impacts  on the large molecular machine remain unstudied.


 \clearpage 
 
 \section{Experimental question}
 
We hypothesize that phosphorylation of the \gct IDR is a key event in the regulation of microtubule dynamics which allows cells precise control over the building of the mitotic spindle. Moreover we propose that such control is achieved via the local  addition of negative charge modulates the global dynamics and conformational sampling of the \gct.
 
 
 \section{Approach}
 

In order to study changes in conformational sampling of the \gct, we use a powerful computational technique known as Molecular Dynamics (MD) simulations. We will be simulating the dynamics of two forms of the \gct: WT, and Y11D (Y445 in the full protein). We will perform simulations on the \gct in isolation as well as in the presence of the entire $\gamma$-Tubulin protein. Analysis of MD simulations will be guided and validated by NMR experiments on the same system performed by collaborators.

 
 
 
 
 